server/lib/backyourstack/dispatcher.js

```diff
@@ -148,3 +149,31 @@ export async function dispatchFunds(order) {
     { concurrency: 3 },
   );
 }
+
+export async function dispatch(order, subscription) {
+  try {
+    const dispatchedOrders = await dispatchFunds(order);
+    // update subscription next dispatch date
+    if (dispatchedOrders) {
+      const currentDispatchDate = (subscription.data && subscription.data.nextDispatchDate) || new Date();
+      subscription.data = {
+        nextDispatchDate: getNextDispatchingDate(subscription.interval, currentDispatchDate),
+      };
+      await subscription.save();
+    }
+    const collective = await models.Collective.findByPk(order.FromCollectiveId);
+    // send confirmation email to collective admins
+    await models.Activity.create({
+      type: activities.BACKYOURSTACK_DISPATCH_CONFIRMED,
+      UserId: order.CreatedByUserId,
+      CollectiveId: collective.id,
+      data: {
+        orders: dispatchedOrders,
+        collective: collective.info,
+      },
+    });
+  } catch (err) {
+    console.log('>>>> Background dispatch failed');
+    console.error(err);
```

Comment:
What do you plan to do if the dispatch is failing and we end up here in production?

Reply:
I was about to ask for suggestion on Slack, I'm thinking we can implement a way to retry dispatch just like we have for subscriptions, I also think in a situation where parts of the fund has been dispatched and failure occurred halfway, we don't want to retry that process instead have human investigation, having a real time alert system like "Sentry"  might be helpful. Another challenge will be how to determine what failure to retry.

Reply:
Can you propose a plan, short-term, mid-term, long-term? So I can give input on it?